QEfficient/cloud/export.py

```diff
@@ -71,8 +77,13 @@ def main(
     :hf_token: str. HuggingFace login token to access private repos.
     :local_model_dir: str. Path to custom model weights and config files.
     """
-    cache_dir = check_and_assign_cache_dir(local_model_dir, cache_dir)
-    get_onnx_model_path(model_name=model_name, cache_dir=cache_dir, hf_token=hf_token, local_model_dir=local_model_dir)
+    get_onnx_model_path(
```

Comment:
why did you remove `check_and_assign_cache_dir` ?

Reply:
Had picked up this change from Release 1.16 branch https://github.com/quic/efficient-transformers/pull/69/files